### PR TITLE
Deploy in local Tomcat

### PR DESCRIPTION
**Pull request by reshetnev**
Reveiw, please.

### CODE REVIEW

File: tomcat_deploy.bat [hunk 1]
```diff
@@ -0,0 +1 @@
+gradle deployToTomcat
```
Comment (DzmitryShylovich) [inline tomcat_deploy.bat]: I guess we don't need external bat files to run gradle tasks.

File: build.gradle [hunk 2]
```diff
@@ -146,3 +146,25 @@ check.dependsOn integrationTest
...
+
+//---------------------------Deploy in local Tomcat--------------------------------------------------//
+
+def tomcat_home = System.getenv('CATALINA_HOME')
+def tomcat_webapps = tomcat_home+'/webapps'
+def tomcat_bin = tomcat_home+'/bin'
+
+task deployToTomcat(type: Copy) {
+    from war
+    into tomcat_webapps
```
Comment (DzmitryShylovich) [inline build.gradle]: Don't upload the war with a .war extension. Tomcat will attempt to start it before it's done copying, fail, and you will have to restart tomcat.
So u have 2 options:
1) You can call stopTomcat task before coping war file and call startTomcat after the copping will be done.
2) If tomcat instance have already running u can upload war in some tmp file({warName}.war.new). Remove old war file. Activate new war by removing `.new` suffix. More info [here](http://cadrlife.blogspot.com/2013/07/simple-war-deploy-to-tomcat-with-gradle.html).

**Comment by DzmitryShylovich** [comment]
We're using [GitFlow](http://datasift.github.io/gitflow/) branching model. According to it all branches must have 'feature' preffix.

File: tomcat_deploy.bat (hunk 1 above)
Comment (reshetnev) [inline tomcat_deploy.bat]: I agree, but it for convenience...

File: build.gradle (hunk 2 above)
Comment (reshetnev) [inline build.gradle]: I understood.

File: tomcat_deploy.bat (hunk 1 above)
Comment (DzmitryShylovich) [inline tomcat_deploy.bat]: 1. Always add empty lines at the end of the files. U can configure your IDE to do it automatically.
2. The main intention of script files is to aggregate some logic which u don't want to type manually every time. As this file contains only 1 line there's no difference between `gradle dT` or `tomcat_deploy.bat.`

Comment (reshetnev) [inline tomcat_deploy.bat]: Ok

**Comment by reshetnev** [comment]
Branch has updated.
Reveiw, please.

**Pull request merged**